### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Link a Cluster-Link a Cluster_HD Insight_Keyboard: Not able to access 'Soft wrap', 'Scroll to end' and 'Clear all' controls using keyboard

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3610]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.1.1

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS2.1.1

**Note: Similar issue is observed in** 
1.Link a Cluster-Link a Cluster_Livy Service
2.Link SQL Server big data cluster

**Repro Steps:**
1)Launch IntelliJ IDEA in (Darcula, High contrast, IntelliJ theme)
2)Tab till Create New Project button and press enter
3)Tab till view-->Tool Window-->Azure Explorer and press enter
4)Tab till signin-->copy and open & sign in to azure
5)Tab till select button and press enter
6)Tab till HDInsight and open context menu by using keyboard key
7)Tab till link a cluster and press enter
8)Verify whether 'Soft wrap', 'Scroll to end' and 'Clear all' controls are accessible using keyboard

**Actual Result:**
Not able to access 'Soft wrap', 'Scroll to end' and 'Clear all' controls using keyboard

**Expected Result:**
'Soft wrap', 'Scroll to end' and 'Clear all' controls should be accessible using keyboard

**User Impact:**
Keyboard only users will find difficulty if controls are not accessible using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D&cid=80b01241-379c-4451-a23a-eb7e19dec151

![Bug_Not able to access controls using keyboard](https://user-images.githubusercontent.com/51435266/65499187-d9cf7780-deda-11e9-906a-317a5fd6e612.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749710
